sandbox/registration-processor-mz.properties

```diff
@@ -625,6 +624,10 @@ mosip.regproc.abis.handler.server.port=9071
 mosip.regproc.abis.handler.eventbus.port=5726
 mosip.regproc.abis.handler.server.servlet.path=/registrationprocessor/v1/abishandler
 
+mosip.regproc.abis.handler.biometric-modalities-segments-mapping={'Finger': {'Left Thumb','Left LittleFinger','Left IndexFinger','Left MiddleFinger','Left RingFinger','Right Thumb','Right LittleFinger','Right IndexFinger','Right MiddleFinger','Right RingFinger'},'Iris':{'Left','Right'},'Face':{'Face'}}
+mosip.regproc.abis.handler.biometric-segments-exceptions-mapping={'Left Thumb' : 'leftThumb','Right Thumb' : 'rightThumb','Left MiddleFinger' : 'leftMiddle','Left RingFinger' : 'leftRing','Left LittleFinger' : 'leftLittle','Left IndexFinger' : 'leftIndex','Right MiddleFinger' : 'rightMiddle','Right RingFinger' : 'rightRing','Right LittleFinger' : 'rightLittle','Right IndexFinger' : 'rightIndex','Left' : 'leftEye','Right' : 'rightEye','Face' : 'face'}
+
```

Comment:
Please remove the below 2 additional lines

Reply:
Incorporated review comments